commands/permissions.go

```diff
@@ -44,11 +46,37 @@ var ShowRoleCmd = &cobra.Command{
 	RunE:    withClient(showRoleCmdF),
 }
 
+var AssignUsersCmd = &cobra.Command{
+	Use:   "assign [role_name] [username...]",
+	Short: "Assign users to role (EE Only)",
+	Long:  "Assign users to a role by username (Only works in Enterprise Edition).",
+	Example: `  permissions assign system_admin john.doe jane.doe
```

Comment:
nit: could you please add some comments to the examples? something like what we have in the [plugin marketplace ones](https://github.com/mattermost/mmctl/blob/master/commands/plugin_marketplace.go#L40)

Reply:
Added some comments.